docs/core/runtime-config/garbage-collector.md

```diff
@@ -771,3 +771,16 @@ Example *app.config* file:
 
 > [!TIP]
 > Experiment with different numbers to see which value works best for you. Start with a value between 5 and 7.
+
+## Dynamic adaptation to application sizes (DATAS)
+
+- Configures the garbage collector to use DATAS. DATAS adapts to application memory requirements, meaning the app heap size should be roughly proportional to the long-lived data size.
+- Enabled by default starting in .NET 9.
+
+|                          | Setting name               | Values    | Version introduced |
+|--------------------------|----------------------------|-----------|--------------------|
+| **runtimeconfig.json**   | `System.GC.DynamicAdaptationMode` | `1` - enabled<br/> `0` - disabled | .NET 8 |
+| **Environment variable** | `DOTNET_GCDynamicAdaptationMode`  | `1` - enabled<br/> `0` - disabled | .NET 8 |
+| **MSBuild property** | `GarbageCollectionAdaptationMode` | `1` - enabled<br/> `0` - disabled | .NET 8 |
```

Comment:
The following order makes more sense to me. The ENV approach is the more temporary of the three and msbuild and runtimeconfig are the same thing in the sense that the former is the convenience version of the latter (so they should be next to one another in that order).

```suggestion
| **Environment variable** | `DOTNET_GCDynamicAdaptationMode`  | `1` - enabled<br/> `0` - disabled | .NET 8 |
| **MSBuild property** | `GarbageCollectionAdaptationMode` | `1` - enabled<br/> `0` - disabled | .NET 8 |
| **runtimeconfig.json**   | `System.GC.DynamicAdaptationMode` | `1` - enabled<br/> `0` - disabled | .NET 8 |
```